src/filters/frame_skipping_pp.py

```diff
@@ -0,0 +1,52 @@
+from src.filters.abstract_pp import AbstractPP
+from src.utils import image_utils
+from typing import List
+
+
+class frameSkippingPP(AbstractPP):
+
+    def __init__(self, threshold=0.0, compare_foreground=False, 
+                 distance_metric='absolute_difference'):
```

Comment:
string parameter, eg. 'absolute_difference' might cause confusion. 

This is not a major issue, as many python packages (sklearn etc.) take string params as input. For Eva, it would be good if we follow strong typing. We can make distance metric an *Enum* containing different metrics instead of strings.

PS: You will still be able to use *getattr* even with enums

Reply:
Made distance metric an ENUM.